scripts/qemu/configs/clx_2_var1.xml

```diff
@@ -0,0 +1,37 @@
+<!-- SPDX-License-Identifier: BSD-2-Clause
+# Copyright (C) 2020 - 2024 Intel Corporation.
+-->
+<domain type='qemu'>
+  <name>CascadeLake2Var1</name>
```

Comment:
we could think of some meaningful name for these - I know it was based on memkind's names but we are not using CLX capabilities here

Reply:
done